.changeset/rare-snails-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+"docs": patch
```

Comment:
This _might_ cause some issues because we've [added docs to the ignored packages for changeset](https://github.com/FuelLabs/fuels-ts/pull/1421/files#diff-ed5b41335ff968c5cfd8035d4b8f8c8c0538b6746182d522adb0312eb9137fc5R12)